.circleci/workflows.yml

```diff
@@ -376,7 +376,29 @@ jobs:
           name: Unit tests
           command: yarn test:unit
 
-  build-and-publish-container-image:
+  check-pipeline:
+    executor: vm
+    steps:
+      - initialize-environment
+      - run:
+          name: Check pipeline
+          command: poetry run python scripts/check-pipeline.py $CIRCLE_WORKFLOW_ID
```

Comment:
We could have something like
```
  environment:
    PIPELINE_ID: << pipeline.id >>
```
and then
```
          command: poetry run python scripts/check-pipeline.py $PIPELINE_ID
```

Reply:
will try this but not sure if this is valid